Assets/LeapMotion/Scripts/DataStructures/Pool.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Leap.Unity {
+
+  public interface IPoolable {
```

Comment:
It looks like the way to implement an IPoolable<T> is pretty simple: Implement OnSpawn(), use Pool<T>.Spawn() to get a T, and then when a given T is ready to be recycled, just call Pool<T>.Recycle(theT). There aren't any other gotchas to think about, right? If so, (1) that's super sweet, (2) could you put a doc comment here for that so that an implementer is told that explicitly?